components/org.wso2.carbon.identity.oauth.ciba/pom.xml

```diff
@@ -0,0 +1,155 @@
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-Package>
+                            org.wso2.carbon.identity.oauth.ciba.internal,
+                        </Private-Package>
+                        <Import-Package>
+                            javax.servlet.http.*; version ="${imp.pkg.version.javax.servlet}",
+                            javax.servlet.*;version ="${imp.pkg.version.javax.servlet}",
+
+                            org.apache.commons.logging.* ; version="${commons-logging.osgi.version.range}",
+                            org.apache.felix.scr; version = "${apache.felix.scr.ds.annotations.version}",
+                            org.apache.oltu.oauth2.* ; version = "${oltu.package.import.version.range}",
+
+                            net.minidev.json.*; version="${json-smart.version}",
+
+                            com.nimbusds.* ; version ="${nimbusds.osgi.version.range}",
+
+                            org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}",
+                            org.osgi.service.component; version="${osgi.service.component.imp.pkg.version.range}",
+
+                            org.wso2.carbon.identity.oauth2.*; version ="${project.version}",
+                            org.wso2.carbon.identity.oauth.* ; version ="${project.version}",
+                            org.wso2.carbon.registry.core.* ; version ="${carbon.kernel.version}",
+                            org.wso2.carbon.user.core.* ; version ="${carbon.kernel.version}",
+                            org.wso2.carbon.user.api.* ; version ="${carbon.kernel.version}",
+                            org.wso2.carbon.identity.base.* ; version = "${carbon.identity.framework.version}",
+                            org.wso2.carbon.identity.openidconnect.model.* ; version = "${project.version}",
+                            org.wso2.carbon.identity.application.authentication.framework.model.* ; version
+                            ="${carbon.identity.framework.version}",
+                            org.wso2.carbon.identity.core.*; version = "${carbon.identity.framework.version}",
+
+
+                        </Import-Package>
+                        <Export-Package>
+                            !org.wso2.carbon.identity.oauth.ciba.internal,
+                            org.wso2.carbon.identity.oauth.ciba.*;
+                            version="${identity.inbound.auth.oauth.exp.pkg.version}",
+                        </Export-Package>
+                    </instructions>
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-resources-plugin</artifactId>
```

Comment:
Group ID missing, version need to be moved to the parent pom

Reply:
none of the other poms in INBOUND component has group ID.